src/main/java/seedu/address/ui/ResultDisplay.java

```diff
@@ -20,7 +23,37 @@ public ResultDisplay() {
         super(FXML);
     }
 
-    public void setFeedbackToUser(String feedbackToUser) {
+    /**
+     * Sets the command box style to use the default style.
+     */
+    private void setStyleToDefault() {
+        resultDisplay.getStyleClass().remove(ERROR_STYLE_CLASS);
+    }
+
+    /**
+     * Sets the command box style to indicate a failed command.
+     */
+    private void setStyleToIndicateCommandFailure() {
+        ObservableList<String> styleClass = resultDisplay.getStyleClass();
+
+        if (styleClass.contains(ERROR_STYLE_CLASS)) {
+            return;
+        }
+
+        styleClass.add(ERROR_STYLE_CLASS);
+    }
+
+    public void setFeedbackErrorToUser(String feedbackToUser) {
+        setStyleToIndicateCommandFailure();
+        setFeedbackToUser(feedbackToUser);
+    }
+
+    public void setFeedbackSuccessToUser(String feedbackToUser) {
+        setStyleToDefault();
+        setFeedbackToUser(feedbackToUser);
+    }
+
+    private void setFeedbackToUser(String feedbackToUser) {
```

Comment:
I would inline this to the two methods above to reduce the complexity of the code.